docs/concepts/vega-chain/data-nodes.md

```diff
@@ -0,0 +1,139 @@
+* Staking rewards per epoch per Vega ID of validators/tokenholders, including the asset, amount, percentage of total and timestamp
+* Governance proposals: All proposals submitted, the 'yes' and 'no' votes, and the parameter changes proposed
+* Trading related data: Prices current and historical such as best bid/ask, mid, mark price; open interest, trade price/volume, closeout trades, loss socialisation events, position mark-to-market events; indicative auction uncrossing price and volume
+* Market lifecycle events: Auction start, end, reason for entering, type of auction; settlement, trading terminated
+* Liquidity provision data: liquidity commitment order submissions, changes in equity-like share, target stake, supplied stake
+* Risk data: Margin level events, Model parameter changes, Risk factor changes, Price monitoring bound setting changes, Price monitoring bounds
+* Candle data: Data that corresponds to trades during a certain time period: first trade, highest traded price, lowest traded price
+
+## APIs
+For clients to communicate with data nodes, the protocol exposes a set of APIs and methods for reading data. Currently there are three protocols to communicate with the data node APIs: GraphQL, gRPC, and REST. 
+
+Querying for data provided through the APIs is the easiest way to access the information that the data node provides, whether that's directly, or by using a dApp/user interface.
+
+If you are running your own data node, you can choose to enable any/all of the protocols, to tailor to your needs. Data nodes run by validators are expected to provide GraphQL, gRPC, and REST, and reliably serve data.
+
+## Database
+The data node relies on PostgreSQL and IPFS. Postgres, an open source, relational database that supports relational and non-relational queries, is used to store and provide the data.
+
+IPFS is used to share decentralised history segments across a data node's network.
+
+:::note Go deeper
+* [About PostgreSQL ↗](https://www.postgresql.org/about/): Read about PostgreSQL and explore the documentation.
+* [About IPFS ↗](https://docs.ipfs.tech/concepts/what-is-ipfs/): Get familiar with IPFS.
+:::
+
+## Connect directly to database
+It's possible to connect directly to the Postgres database of a data node without running a data node yourself, if the data node you're using allows it. You'll need the connection details from the data node operator, such as the relevant address, username and password in order to connect directly to that node's database.
+
+Alternatively, you can run your own data node and use [decentralised history](#decentralised-history) to pull existing data into your new database, or start fresh.
+
+:::caution Reliability
+PostgreSQL database schemas are liable to change without warning and without deprecation.
+:::
+
+<!-- Link to the data node postgres information.-->
+
+## Data retention
```

Comment:
@guoguojin Does anything in this section need to change based on https://github.com/vegaprotocol/vega/pull/7476?

Reply:
Yes, you're missing a few data types:

trades_candle_1_day = 1 year
markets = 1 year
deposits = 1 year
withdrawals = 1 year
blocks = equal to the longest retention duration across all tables

The default retention period for lite = 1 day

Retention policy for any entity can be overridden by creating an entry in the config file under the SQLStore section:

```yaml
...
[SQLStore]
  [[SQLStore.RetentionPolicies]]
    HypertableOrCaggName = "balances"
    DataRetentionPeriod = "7 days"

  [[SQLStore.RetentionPolicies]]
    HypertableOrCaggName = "checkpoints"
    DataRetentionPeriod = "7 days"

...
```

Reply:
Also, the retention policy for each entity is logged out when the node starts so they should be able to see what it is easily from there.

If they want to change the retention policy, a restart of the data-node is required.